### ISSUE TITLE
TS breakpoints no longer work in VS Code  after migrating Angular from 7.0 to 8.1

### ISSUE THREAD

**Issue by alincomsa** [angular/angular#31502]
# 🐞 bug report

### Description

TS breakpoints no longer work after upgrading to 8.1

## 🔬 Minimal Reproduction
Environment used: VS Code
generate an angular 7 app, set up the standard debug configuration such as below, breakpoints get hit. Then upgraded to Angular 8.1, the breakpoints no longer get hit.

            "name": "Launch Chrome against localhost",
            "url": "http://localhost:4200",
            "webRoot": "${workspaceFolder}"

**Comment by JoostK** [comment]
I think this has the same root cause as #31417, i.e. the VSCode language service integration is broken. I will therefore close this as duplicate, but if you continue to experience issues after #31417 has been resolved please open a new issue.
